importer/load.py

```diff
@@ -50,8 +50,8 @@ def main(argv):
                         commands.append("SET article.ISSN = '{0}'"
                                         .format(record["ISSN"]))
                     if "Author" in record:
-                        commands.append("MERGE (author:Author {{name:'"
-                                        "{0}'}}) MERGE (article)-"
+                        commands.append("MERGE (author:Author {{name:\""
+                                        "{0}\"}}) MERGE (article)-"
```

Comment:
So the one set are to split the line in Python to meet line length requirements, and the other is the escaped " character as need to use that instead of '  otherwise it breaks on errors like the name:
Jake N'elson